Mathlib/Algebra/AddTorsor/Basic.lean

```diff
@@ -76,6 +76,14 @@ theorem vsub_sub_vsub_cancel_left (p₁ p₂ p₃ : P) : p₃ -ᵥ p₂ - (p₃
 theorem vadd_vsub_vadd_cancel_left (v : G) (p₁ p₂ : P) : (v +ᵥ p₁) -ᵥ (v +ᵥ p₂) = p₁ -ᵥ p₂ := by
   rw [vsub_vadd_eq_vsub_sub, vadd_vsub_assoc, add_sub_cancel_left]
 
+theorem vadd_vsub_vadd_comm (v₁ v₂ : G) (p₁ p₂ : P) :
+    (v₁ +ᵥ p₁) -ᵥ (v₂ +ᵥ p₂) = (v₁ - v₂) + (p₁ -ᵥ p₂) := by
+  rw [vsub_vadd_eq_vsub_sub, vadd_vsub_assoc, add_sub_assoc, ← add_comm_sub]
+
+theorem vadd_vsub_vadd_comm_rev (v₁ v₂ : G) (p₁ p₂ : P) :
+    (v₁ +ᵥ p₁) -ᵥ (v₂ +ᵥ p₂) = (p₁ -ᵥ p₂) + (v₁ - v₂) := by
+  rw [vadd_vsub_vadd_comm, add_comm]
```

Comment:
I meant
```suggestion
theorem sub_add_vsub_comm (v₁ v₂ : G) (p₁ p₂ : P) :
    (v₁ - v₂) + (p₁ -ᵥ p₂) = (v₁ +ᵥ p₁) -ᵥ (v₂ +ᵥ p₂) :=
  vadd_vsub_vadd_comm _ _ _ _ |>.symm
```

Reply:
No, in general we don't want both directions; I only suggest we have one here because we have both directions of the `+` / `-` version of this statement